stable/kubernetes-dashboard/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.8.0
+version: 1.8.1
```

Comment:
The PR looks good to me, except its a new feature rather then just a bugfix, so should be 1.9.0.

Reply:
@kfox1111 thank you for the quick turn around. Just amended it.